horizons/gui/widgets/minimap.py

```diff
@@ -753,14 +760,32 @@ def iter_points(self):
 				x = x_i + location.left
 				# TODO: better name
 				asdf = int(abs(x / location.width - 0.5) * location.height)
-				for y_i in range(asdf, location.height - asdf):
+				for y_i in range(self.get_min_y(x), self.get_max_y(x)):
```

Comment:
Good change. The earlier code wasn't very clear. You can remove `asdf ` entirely now, I think.

Reply:
You're right! I didn't even realize.